.github/workflows/super-linter.yml

```
@@ -11,9 +11,9 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Lint code base
-        uses: docker://github/super-linter:v2.2.2
+        uses: docker://github/super-linter:v4
         env:
           # Listed but disabled linters would be nice to have.
```

```suggestion
        uses: docker://github/super-linter:v4
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          # Listed but disabled linters would be nice to have.
```

Possible solution to super-linter error.